src/index.html

```diff
@@ -5,9 +5,397 @@
+              <input
+                type="text"
+                name="name"
+                placeholder="Name"
+                id="input-name"
+                class="input form__input"
+              >
+            </label>
+            <label for="input-phone">
+              <input
+                type="text"
+                name="phone"
+                placeholder="Phone"
+                id="input-phone"
+                class="input form__input"
+              >
+            </label>
+            <label for="input-email">
+              <input
+                type="text"
+                name="email"
+                placeholder="Email"
+                id="input-email"
+                class="input form__input"
+              >
+            </label>
+            <button class="button form__button">
+              Consultation
+            </button>
+          </form>
+        </div>
+      </section>
+      <section id="testimonials" class="section">
+        <h2 class="heading testimonials__top">
+          Testimonials
+          <span class="heading__crown">
+          </span>
+        </h2>
+        <div class="testimonials">
+          <div class="person">
+            <img
+              src="images/people/1.png"
+              alt="Leslie Mckinney photo"
+              class="person__image"
+            >
+            <h3 class="person__name">
+              Leslie Mckinney
+            </h3>
+            <p class="person__position">
+              CEO
+            </p>
+            <p class="person__text">
+              Nice work, Certificates National Association of Realtors (USA)
+            </p>
+          </div>
+          <div class="person">
+            <img
+              src="images/people/2.png"
+              alt="Scarlett Hawkins photo"
+              class="person__image"
+            >
+            <h3 class="person__name">
+              Scarlett Hawkins
+            </h3>
+            <p class="person__position">
+              Senior Vice President
+            </p>
+            <p class="person__text">
+              My soul is illuminated by an unearthly joy, as these beautiful spring morning, which I enjoy with all my heart
+            </p>
+          </div>
+          <div class="person">
+            <img
+              src="images/people/3.png"
+              alt="Max Henry photo"
+              class="person__image"
+            >
+            <h3 class="person__name">
+              Max Henry
+            </h3>
+            <p class="person__position">
+              Chief Design Officer
+            </p>
+            <p class="person__text text--3">
+              When my lovely valley of steam rises and half-day sun is above an impermeable
+            </p>
+          </div>
+        </div>
+      </section>
+      <section id="contacts" class="section">
+        <h2 class="heading contact__top">
+          Contact us
+          <span class="heading__crown">
+          </span>
```

Comment:
пустые тэги в этом случае можно в одну строку записать
```suggestion
          <span class="heading__crown"></span>
```